src/ui/Quickview/Quickview.ts

```diff
@@ -453,10 +464,64 @@ export class Quickview extends Component {
         body: this.element.ownerDocument.body,
         sizeMod: 'big'
       });
+      this.makeModalboxAccessible(this.modalbox.modalBox);
       return computedModalBoxContent;
     });
   }
 
+  private getFocusableElements(container: HTMLElement) {
+    return _.sortBy(container.querySelectorAll<HTMLElement>('[tabindex]'), element => element.tabIndex);
+  }
+
+  private createFocusTrap(element: HTMLElement) {
```

Comment:
Was creating such a focusTrap the recommended approach on dequeue university ?

Also, this whole function is pretty hard to follow/read.

Id split/break the logic up in different method. 
I could even see all of this moved to `FocusTrap` module/class maybe ?

Which accept a `container` or root element inside which the trap should be created, and the tab cycling logic would be implemented.

Reply:
I couldn't find any recommendation from Deque other than to make sure elements outside of a modal aren't focusable.

> If a screen reader user navigates by elements such as headings, landmarks, links, form elements, etc., the only elements available when the dialog is open are the elements in the dialog itself. There must not be a way to access anything outside of the dialog with screen reader keyboard shortcuts.

As for the code being hard to read, I agree that it could be broken up into a separate class and I'll do that.